cpp/include/Ice/LocalException.h

```diff
@@ -3643,7 +3643,8 @@ class ICE_CLASS(ICE_API) FixedProxyException : public LocalExceptionHelper<Fixed
 };
 
 /**
- * Indicates that the response to a request has already been sent; re-dispatching such a request is not possible.
+ * Indicates the application attempted to call the callbacks for an async dispatch more than once, or called an async
```

Comment:
I don't think this exception and `Incoming::setResponseSent` are still needed since afaict, it's no longer possible to occur.

Reply:
As the doc-comment describes, this can definitely occur, and we even have tests checking it.

It's always a logic error - a bug in the application. For example, if the application calls the response _and_ the exception callbacks for the same dispatch.

Reply:
We could rename the exception to make the logic error cleaner. Any suggestion?